haproxy/meta/heka.yml

```diff
@@ -99,6 +99,10 @@ aggregator:
 {%- if listen.get('type', None) == 'openstack-service' %}
       - {{ listen_name }}_http_errors
 {%- endif %}
+{%- if listen.service_name is defined %}
+      dimension:
+        service: {{ listen.service_name }}
```

Comment:
can you fallback to {{ listen.get('service_name', listen_name) }} ? does it make sense here?